docs/src/ci-intro-js.md

```diff
@@ -0,0 +1,103 @@
+---
+id: ci-intro
+title: "CI Github Actions"
+---
+
+When installing Playwright you are given the option to add a GitHub actions. This creates a `playwright.yaml` file inside a `.github/workflows` containing everything you need so that your tests run on each push or pull request.
+
+**What you will learn:**
+
+- [How to use GitHub actions to run your tests](#github-actions)
+- [How to create a repo and push to GitHub](#create-a-repo-and-push-to-github)
+- [How to open the workflows](#opening-the-workflows)
+- [How to view the test Logs](#viewing-test-logs)
+- [How to download the Playwright Report from GitHub](#downloading-the-playwright-report)
+- [How to view the Playwright Report](#viewing-the-playwright-report)
+- [How to view the trace](#viewing-the-trace)
+
+## GitHub Actions
+
+Tests will run on push or pull request on branches main or master. The workflow will install all dependencies, install playwright and then run the tests. It will also create the playwright report.
+
+```yaml
+name: Playwright Tests
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.x"
+      - name: Install dependencies
+        run: npm ci
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
+      - name: Run Playwright tests
+        run: npx playwright test
+      - uses: actions/upload-artifact@v2
+        if: always()
+        with:
+          name: playwright-report
+          path: playwright-report/
+          retention-days: 30
+```
+
+### Create a Repo and Push to GitHub
+
+[Create a repo on GitHub](https://docs.github.com/en/get-started/quickstart/create-a-repo) and create a new repository or push an existing repository. Follow the instructions on GitHub and don't forget to initialize a git repository using the `git init` command so you can push your code.
```

Comment:
I feel like the linked guide teaches only half of that what's required. Maybe something from here fits? https://github.com/git-guides/git-push